Gemfile

```diff
@@ -27,6 +27,8 @@ end
 
 gem 'rake', :require => false
 gem 'facter', ENV['FACTER_GEM_VERSION'], :require => false, :groups => [:test]
+# https://github.com/voxpupuli/facterdb/pull/244#issuecomment-1105120825
+gem 'facterdb', '= 1.15.0', :require => false, :groups => [:test]
```

Comment:
can you please undo this? We were able to figure out a proper fix for rspec-puppet-facts and released it a few minutes ago.